src/repository/models.py

```diff
@@ -533,6 +539,11 @@ def download_url(self):
         )
 
 
+class PreprintSupplementaryFile(models.Model):
+    preprint = models.ForeignKey(Preprint)
```

Comment:
This would do the job if we go for a one-to-many relatioship. It's not needed on a M2M, since django will create the intermediate table and keys automatically